### PR TITLE
WAZO-3239 Support archive mirrors for stretch upgrade path

### PR DESCRIPTION
**Pull request by DrPyser**
Will require backporting to enable upgrade path from 19.12 stacks.

### CODE REVIEW

File: bin/wazo-dist-upgrade-buster [hunk 1]
```diff
@@ -124,6 +124,33 @@ EOF
...
+EOF
+    elif [ ${from} == 'buster' ] && [ ${to} == 'stretch' ]; then
+        sed -i "/\(deb\.debian\.org.*) buster.*/d" /etc/apt/sources.list
+        sed -i "/\(security\.debian\.org.*\) buster.*/d" /etc/apt/sources.list
+        cat >> /etc/apt/sources.list <<EOF
+deb http://archive.debian.org/debian-archive/debian/ stretch main
+deb-src http://archive.debian.org/debian-archive/debian/ stretch main
+
+deb http://archive.debian.org/debian-archive/debian/ stretch-backports main
+deb-src http://archive.debian.org/debian-archive/debian/ stretch-backports main
```
Comment (fblackburn1) [inline bin/wazo-dist-upgrade-buster]: We do not support backport

File: bin/wazo-dist-upgrade-buster [hunk 2]
```diff
@@ -124,6 +124,33 @@ EOF
...
+deb-src http://deb.debian.org/debian buster main
+
+deb http://security.debian.org/debian-security buster main
+deb-src http://security.debian.org/debian-security buster main
+
+deb http://deb.debian.org/debian buster-updates main
+deb-src http://deb.debian.org/debian buster-updates main
+
+deb http://deb.debian.org/debian buster-backports main
+deb-src http://deb.debian.org/debian buster-backports main
```
Comment (fblackburn1) [inline bin/wazo-dist-upgrade-buster]: We do not support backport

**Review by fblackburn1 — CHANGES_REQUESTED** [review]

File: bin/wazo-dist-upgrade-buster [hunk 3]
```diff
@@ -124,6 +124,33 @@ EOF
...
+EOF
+    elif [ ${from} == 'buster' ] && [ ${to} == 'stretch' ]; then
+        sed -i "/\(deb\.debian\.org.*) buster.*/d" /etc/apt/sources.list
+        sed -i "/\(security\.debian\.org.*\) buster.*/d" /etc/apt/sources.list
+        cat >> /etc/apt/sources.list <<EOF
+deb http://archive.debian.org/debian-archive/debian/ stretch main
+deb-src http://archive.debian.org/debian-archive/debian/ stretch main
```
Comment (fblackburn1) [inline bin/wazo-dist-upgrade-buster]:
```suggestion
deb http://archive.debian.org/debian/ stretch main
deb-src http://archive.debian.org/debian/ stretch main
```

Comment (fblackburn1) [reply bin/wazo-dist-upgrade-buster]: Oh in fact this line work, but I copied the message for the announce (and the `debian/` part was missing)

Comment (DrPyser) [reply bin/wazo-dist-upgrade-buster]: Oh, you're right `/debian-archive` is not required.

**Review by fblackburn1 — APPROVED** [review]
Thank you
But be careful to always test your bash script in real situation

**Pull request merged**